packages/formspree-react/src/context.tsx

```diff
@@ -1,98 +1,75 @@
-import React, { Suspense, lazy, useContext, useEffect, useState } from 'react';
-import { createClient, getDefaultClient } from '@formspree/core';
-import type { Client, Config } from '@formspree/core';
+import { createClient, getDefaultClient, type Client } from '@formspree/core';
+import { Elements } from '@stripe/react-stripe-js';
 import { loadStripe } from '@stripe/stripe-js/pure.js';
-import type { Stripe } from '@stripe/stripe-js';
-
-const Elements = lazy(() =>
-  import('@stripe/react-stripe-js').then((module) => {
-    return { default: module.Elements };
-  })
-);
-
-export type FromspreeContextType = {
+import React, {
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+  type ReactNode,
+} from 'react';
+
+export type FormspreeContextType = {
   client: Client;
 };
 
 export type FormspreeProviderProps = {
+  children: ReactNode;
   project?: string;
-  children: React.ReactNode;
   stripePK?: string;
 };
 
-const FormspreeContext = React.createContext<FromspreeContextType | null>(null);
-
-FormspreeContext.displayName = 'Formspree';
-
-let stripePromise: Promise<Stripe | null>;
-
-const getStripe = (stripeKey: string) => {
-  if (!stripePromise) {
-    stripePromise = loadStripe(stripeKey);
-  }
-  return stripePromise;
-};
```

Comment:
Don't need this. `@stripe/stripe-js` already ensure not to load stripe script if it is already loaded.

We only need to make sure that we don't call `loadStripe` multiple times (see https://github.com/formspree/formspree-js/pull/48/files#diff-acd46b99f23f96931c115e5f3dd6159a1a4ae7fcb4421cc3c936d9647a1a7e05R31-R33) which will re-instantiate `stripe` client object.